resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.service.erb

```diff
@@ -12,6 +12,9 @@ Restart=on-failure
 StartLimitBurst=5
 #set default privileges to -rw-r-----
 UMask=027
+<% if EZBake::Config[:open_file_limit] -%>
```

Comment:
@janelu2 has a fix that changes the default to "nil", which evals to nil in ruby, which, if unwrapped, should be falsey. That will involve touching less files and seems reasonable to me.